src/ignitus-Shared/ignitus-DesignSystem/ignitus-Atoms/ignitus-defaultTag/styles.ts

```diff
@@ -0,0 +1,62 @@
+import styled from '@emotion/styled';
+import Icon from '../../../ignitus-Utilities/Components/icon';
+
+export const StyledIcon = styled(Icon)`
+  height: 1.2rem;
+  width: 1.2rem;
+  fill: black;
```

Comment:
use black defined as a const in `colors.ts`

Reply:
Done